ts/generated/ffi.WASM_RELEASE_ASYNCIFY.ts

```diff
@@ -161,7 +161,7 @@ export class QuickJSAsyncFFI {
   QTS_BuildIsAsyncify: () => number =
     this.module.cwrap("QTS_BuildIsAsyncify", "number", [])
 
-  QTS_NewFunction: (ctx: JSContextPointer, func_id: number, name: string) => JSValuePointer =
+  QTS_NewFunction: (ctx: JSContextPointer, func_id: uint32_t, name: string) => JSValuePointer =
```

Comment:
looks like the generator is not properly mapping this to `number`. @justjake can you direct me to where that needs to be handled?

Reply:
You can search the generator file for `int` and it should turn up.

Reply:
found, fixed.